dashboard/test/controllers/home_controller_test.rb

```diff
@@ -280,6 +280,60 @@ class HomeControllerTest < ActionController::TestCase
     assert_select '#age-modal', false
   end
 
+  test 'anonymous does not get thank donors dialog' do
```

Comment:
I like to include precondition assertions in tests like this.  Otherwise, future changes to a `setup` block could cause the test to stop testing what it says it's testing.
```suggestion
  test 'anonymous does not get thank donors dialog' do
    assert_nil current_user
```